app-frontend/src/app/pages/projects/detail/exports/exports.html

```diff
@@ -0,0 +1,49 @@
+<div class="list-group">
+    <rf-export-item
+        export="export"
+        ng-repeat="export in $ctrl.exportList track by export.id">
+    </rf-scene-item>
+  </div>
+  <!-- Export List -->
+
+  <!-- Pagination -->
+  <div class="list-group text-center"
+       ng-show="!$ctrl.loading && $ctrl.lastExportResult && $ctrl.lastExportResult.count > $ctrl.lastExportResult.pageSize && !$ctrl.errorMsg">
+    <ul uib-pagination
+        items-per-page="$ctrl.lastExportResult.pageSize"
+        total-items="$ctrl.lastExportResult.count"
+        ng-model="$ctrl.currentPage"
+        max-size="4"
+        rotate="true"
+        boundary-link-numbers="true"
+        force-ellipses="true"
+        ng-change="$ctrl.populateExportList($ctrl.currentPage)">
+    </ul>
+  </div>
+  <!-- Pagination -->
+
+  <!-- No scenes placeholder -->
+  <div ng-if="!$ctrl.loading && $ctrl.lastExportResult && $ctrl.lastExportResult.count === 0"
+       class="list-group">
+    <span class="list-placeholder">You haven't added any scenes to this Project.</span>
+  </div>
+  <!-- No scenes placeholder -->
+
+  <!-- Reload? Export list -->
+  <div class="list-group" ng-show="$ctrl.errorMsg">
+    <span class="list-placeholder">
+      {{$ctrl.errorMsg}}
+      <a ng-click="$ctrl.populateExportList(1)">Try again</a>
```

Comment:
maybe stick a refresh icon in there for good measure 
![image](https://user-images.githubusercontent.com/4392704/27547891-50a2f938-5a65-11e7-8052-b5cba2c0b0d7.png)